### ISSUE TITLE
tough-cookie deprecation notice && publishing

### ISSUE THREAD

**Issue by swashcap** [giakki/uncss#311]
First off, thanks so much for uncss! It’s a fantastic tool.

I’ve been seeing this when installing dependencies in our project:

```shell
$ npm install
# ...
npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/130
```

:scream:

Looks like this comes from uncss’s nested dependency:

```
$ npm ls tough-cookie@2.2.2
# ...
└─┬ uncss@0.14.1
  └─┬ request@2.69.0
    └── tough-cookie@2.2.2 
```

2b67716ab563299f2061f24fedb86aec29afb5e3 addressed the problem by upgrading to `request` → `^2.72.0`, but this hasn’t been published. When are you planning on publishing 0.14.2/0.15.0?

**Comment by RyanZim** [comment]
A major release is planned to include https://github.com/giakki/uncss/pull/307 & https://github.com/giakki/uncss/pull/280. Don't know how soon that'll be.

@mikelambert Do you want to patch release before that?

**Comment by mikelambert** [comment]
Uncss 0.15.0 just published.

**Comment by XhmikosR** [comment]
Isn't this fixed with 0.15?

**Comment by RyanZim** [comment]
@XhmikosR Yep, safe to close IMO.
